staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -58,8 +60,19 @@ var (
 
 	authenticatedAttemptsCounter = metrics.NewCounterVec(
 		&metrics.CounterOpts{
-			Name: "authentication_attempts",
-			Help: "Counter of authenticated attempts.",
+			Name:           "authentication_attempts",
+			Help:           "Counter of authenticated attempts.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
+	)
+
+	authenticationLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Name:           "authentication_latency_seconds",
```

Comment:
I guess this should be "authentication_duration_seconds" now. https://github.com/kubernetes/kubernetes/pull/74418

Reply:
Fine.